client/webserver/site/src/js/register.js

```diff
@@ -156,12 +133,11 @@ export default class RegistrationPage extends BasePage {
       Doc.show(page.appPWErrMsg)
       return
     }
-    if (seed) this.walletForm.setRegMsg('Your Decred wallet is required')
     this.pwCache.pw = pw
     this.auth()
     app.updateMenuItemsDisplay()
     this.walletForm.refresh()
-    await this.changeForm(page.appPWForm, page.newWalletForm)
+    await this.changeForm(page.appPWForm, page.dexAddrForm)
```

Comment:
I think you need a `this.dexAddrForm.refresh()` here too. If I choose the "remember password" option, the password field is still shown on the dex address form.